.github/workflows/refresh-licenses.yml

```diff
@@ -0,0 +1,41 @@
+# Refreshes the licences directory and creates a PR if there are any changes
```

Comment:
@NiccoloFei I think here is "licenses" :D

Reply:
Fixed.

Reply:
heee nop, still there XD